### ISSUE TITLE
HttpTrigger template references wrong package version

### ISSUE THREAD

**Issue by jeffhollan** [Azure/azure-functions-dotnet-worker#173]
Microsoft.Azure.Functions.Worker.Sdk is set to 1.0.1 instead of 1.0.0

```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>net5.0</TargetFramework>
    <LangVersion>preview</LangVersion>
    <AzureFunctionsVersion>v3</AzureFunctionsVersion>
    <OutputType>Exe</OutputType>
    <_FunctionsSkipCleanOutput>true</_FunctionsSkipCleanOutput>
    <RootNamespace>dotnet_isolated</RootNamespace>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.0.1-preview5" OutputItemType="Analyzer"/>
    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.0.0-preview5" />
  </ItemGroup>
  <ItemGroup>
    <None Update="host.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </None>
    <None Update="local.settings.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
      <CopyToPublishDirectory>Never</CopyToPublishDirectory>
    </None>
  </ItemGroup>
</Project>
```

**Comment by ankitkumarr** [comment]
@jeffhollan, this is intentional. Our newest Sdk package that was just released is `1.0.1-preview5`. Maybe it wasn't available when you tried it.

**Comment by jeffhollan** [comment]
Ah ok great must have been a timing issue / cache. You can close
